ToolsForMaintainersOfTheProjectTemplate/Scintilla_iface_synchronizer/cs.py

```diff
@@ -100,6 +100,7 @@ def printEnumDefinitions(f):
 		if v["FeatureType"] in ["enu"] and name not in ["Keys"]: # for all except excluded enums [conflicting]
 			appendComment(indent, out, v)
 			prefix = v["Value"]
+			out.append(indent + "[Flags]")
```

Comment:
Enums now have the [Flags] Attribute which makes them combinable with other constants of same enum type.